apps/emqx_durable_storage/test/emqx_ds_storage_SUITE.erl

```diff
@@ -27,25 +27,6 @@ opts() ->
 
 %%
 
-t_idempotent_store_batch(_Config) ->
```

Comment:
Well, this doesn't hold, since writing batches to the old generations was a bug.
@keynslug Is this property important for e.g. replication layer?

Reply:
Well, `commit_batch` is still idempotent, since `prepare` adds generation id into the cooked batch. I think splitting the operation should ultimately help with this. There are two obstacles:

1. ra doesn't provide an API to call aux server with a timeout (can be worked around)
2. there's no API get leader from a node that is not part of the shard cluster.